### ISSUE TITLE
Add List of Banned/Suspended Users to Mod Panel

### ISSUE THREAD

**Issue by Hendikins** [Weasyl/weasyl#124]
(Previously filed by Nanuk as GitLab issue 8, Fiz as Redmine issue 609)

The moderator control panel should have a page where moderators can access a list of banned/suspended accounts. This should provide the following information:
-  Username
-  Ban date
-  Username of mod/admin
-  Length / release date or "permanent"

**Comment by Hendikins** [comment]
For reference purposes, per @kfkitsune the following pSQL query will get a list of suspended or banned users:
```
``SELECT userid, login_name, email, settings FROM login WHERE settings ~ 's' OR settings ~ 'b';``
```

**Comment by kfkitsune** [comment]
Resolved by merge of #319.
